bin/sandbox

```
@@ -99,7 +99,11 @@ unbundled bin/rails generate solidus:install \
   --auto-accept \
   $@
 
+unbundled bundle add lookbook listen actioncable --group development,test
```

For some reason, it looks like the gems are not added to the sandbox's `Gemfile`.

---

Is it a silent error or you get anything more specific? 
Also I suggest checking if it's the latest version of bundler, just to be sure.

---

Very silent :slightly_smiling_face: Do you get it added in your sandbox?

---

It is now part of the install generator, does it work for you now?

---

I tried it yesterday and it was there.

---

I tried now and it works, but I left a comment about the approach.